app/_locales/en/messages.json

```
@@ -1603,5 +1603,31 @@
   },
   "zeroGasPriceOnSpeedUpError": {
     "message": "Zero gas price on speed up"
+  },
+  "decryptRequest": {
+    "message": "Decrypt request"
+  },
+  "decrypt": {
+    "message": "Decrypt"
+  },
+  "decryptMessageNotice": {
+    "message": "$1 would like to read this message to complete your action",
+	"description": "$1 is website or dapp name"
+  },
+  "decryptMetamask": {
+    "message": "Decrypt with Metamask"
```

```suggestion
    "message": "Decrypt message"
```
The designs have this as "Decrypt message", which I think works a bit better. The `ru` message will need to be updated as well.